src/library.ts

```diff
@@ -0,0 +1,9 @@
+export * from "./Constants";
```

Comment:
Doesn't this add all these classes to the global scope? If so, we should reconsider this when refactoring later.

Reply:
Indeed, the choices made here will determine what other users of the package will be able to use.  Some things like the CodeEditor might also be useful for end users, so this is up for discussion.

Reply:
You can still make thing available to users without clobbering the global scope by scoping them inside a papyros class or object.